app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    command, source, destination = command.split()
```

Comment:
Code Efficiency: You are assuming that the input command is always correct and it will always have three parts when split. What if the command is not in the correct format? It will lead to an error. You should add error checking to handle this.